core/src/main/scala/anorm/Row.scala

```diff
@@ -9,6 +9,11 @@ trait Row {
   private[anorm] val data: List[Any]
 
   /**
+   * Returns a list of metadata items, one for each column.
+   */
+  lazy val metaDataItems: List[MetaDataItem] = metaData.ms
```

Comment:
Where is the danger in exposing this information?

In any case, is there a safer way to obtain column metadata? Is it possible to accomplish what I presented in #42 safely?